validators/kafka/kafka.go

```diff
@@ -28,8 +29,14 @@ func New(cfg *Config, topics ...string) *Validator {
 		KafkaGroupID:              cfg.GroupID,
 	}
 	for _, topic := range topics {
+		var realizedTopicName string
 		topic = fmt.Sprintf("platform.upload.%s", topic)
-		kv.addProducer(topic)
+		if config.Get().UseClowder {
```

Comment:
We're passing this in as a parameter. Do you have a preference on using this in cfg.Get() vs os.Getenv?

Reply:
I think we should leave calls to getenv in the config module.

Reply:
should probably make it consistent and change the os.Getenv in main.go